bbb-graphql-middleware/config/config.yml

```
@@ -5,7 +5,7 @@ server:
   max_connections: 500
   max_connections_per_session_token: 3
   # Rate limit: maximum number of new connections/clients allowed per second.
-  max_connections_per_second: 100
+  max_connections_per_second: 50
```

```suggestion
  max_connections_per_second: 100
```

Keeping this as 100 by default, more testing to follow. We would welcome another PR (@schrd or @gustavotrott ) setting this to 50, let's just not hold the multiple hasuras PR due to this config.